loadtest.py

```diff
@@ -1,48 +1,40 @@
-"""Molotov testing Tecken.
-
+# Molotov testing Tecken.
 
+'''
 Depends on reading the ./stacks/ directory,
 ./downloading/symbol-queries-groups.csv and ./downloading/socorro-missing.csv
-
-"""
-# import sys; sys.path.append('.')
+'''
 
 import csv
+import molotov
```

Comment:
The reason `molotov` was written further down was because it's good practice to separate Python standard lib and Python third-party libs.